coral/src/domain/requests/requests-transformers.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestsWaitingForApproval } from "src/domain/requests/requests-types";
+import { KlawApiResponse } from "types/utils";
+
+const getRequestsWaitingForApprovalTransformer = (
+  data: KlawApiResponse<"getRequestStatistics">
+) => {
+  const { requestEntityStatistics } = data;
+
+  let requestsWaitingForApproval: RequestsWaitingForApproval = {
```

Comment:
without looking at the code too much my fingers are already nervous because I feel it's a perfect case for using `reduce` 😂  

The implementation is totally fine! Just a random-thought-comment here! :D

Reply:
It is, but I thought this was more legible than reduce ^^